plugins/chat/assets/javascripts/discourse/components/chat-user-avatar.gjs

```diff
@@ -37,19 +38,23 @@ export default class ChatUserAvatar extends Component {
     );
   }
 
+  get userPath() {
+    return userPath(this.args.user.username);
+  }
+
   <template>
     <div
       class={{concatClass "chat-user-avatar" (if this.isOnline "is-online")}}
       data-username={{@user.username}}
     >
       {{#if this.interactive}}
-        <div
-          role="button"
-          class="chat-user-avatar__container clickable"
+        <a
+          class="chat-user-avatar__container"
+          href={{this.userPath}}
           data-user-card={{@user.username}}
         >
```

Comment:
Yes because it will be intercepted here: https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/lib/intercept-click.js#L41